docs/ref/settings.txt

```diff
@@ -2532,6 +2532,11 @@ bandwidth but slows down performance. This is used by the
 :class:`~django.middleware.common.CommonMiddleware` and in the :doc:`cache
 framework </topics/cache>`.
 
+.. deprecated:: 1.11
+
+    This setting is deprecated in favor of the ``ConditionalGetMiddleware``,
+    which sets an ETag regardless of this setting.
```

Comment:
This is a minor point, but in other places the phrase seems to be just "`ConditionalGetMiddleware`" or "the `ConditionalGetMiddleware` middleware". "The `ConditionalGetMiddleware`" reads a bit strange to me. Maybe "...in favor of using `ConditionalGetMiddleware`..."